src/add-to-app/android/project-setup.md

```diff
@@ -292,11 +292,11 @@ include ':app'                                    // assumed existing content
 setBinding(new Binding([gradle: this]))                                // new
 evaluate(new File(                                                     // new
   settingsDir.parentFile,                                              // new
-  'my_flutter/.android/include_flutter.groovy'                         // new
+  'flutter_module/.android/include_flutter.groovy'                         // new
 ))                                                                     // new
 ```
 
-Assuming `my_flutter` is a sibling to `MyApp`.
+Assuming `flutter_module` is a sibling to `MyApp`.
```

Comment:
I know you're not touching this, but reading this sentence in isolation is a bit strange. Is this referring to the previous instruction or the next paragraph?